src/keys/list.ts

```diff
@@ -92,9 +87,7 @@ const listKeymap: { [key: string]: EditorAction } = {
   'Mod-Alt-k': skipCommandTracking(wrapInList(schema.nodes.bullet_list)),
   'Shift-Tab': skipCommandTracking(liftToOuterList(schema.nodes.list_item)), // outdent, same as Mod-[
   Tab: skipCommandTracking(sinkListItem(schema.nodes.list_item)), // indent, same as Mod-]
-  Backspace: skipCommandTracking(
-    chainCommands(undoInputRule, listItemBackward, deleteSelection)
-  ),
+  Backspace: skipCommandTracking(listItemBackward),
```

Comment:
Are you sure this won't break the current backspace behavior for the list_item?

Reply:
It should not, but to stay on the safe side please check it out